nature/styleguide/typography.md

```diff
@@ -32,7 +32,33 @@ Use our font styles consistently by always making sure they reflect the informat
 * Body text and paragraphs at base sizes should not exceed 800px in width
 * We use fluid typography by applying properties for different screen sizes, "Max" and "Min" 
 
-### Headings (Nature journals)
+## Headings
+
+## Write headings in sentence case
+
+For example, “Latest reviews and analysis” not “Latest Reviews and Analysis”.
+
+Sentence case is easier for people to read.
+
+Headings should follow [our content guidelines](/content).
+
+### Make headings meaningful
+
+Use descriptive headings that convey the meaning of the section they refer to. This helps users scan and find the information they need, without having to read all of the content from top to bottom.
+For example, “Publish with us” is clearer than “Information on submission”.
+
+### Use heading levels to communicate page structure
+
+Use semantic heading levels <h1> to <h6> to show the order of importance of the content on a page.
+Only include one <h1> per page, for the main heading.
+
+Don’t skip heading levels. For example, don’t follow an <h2> with an <h4>. Doing this can be confusing, particularly for some assistive technology users.
```

Comment:
```suggestion
Don’t skip heading levels. For example, don’t follow an `<h2>` with an `<h4>`. Doing this can be confusing, particularly for some assistive technology users.
```